DatadogSessionReplay/Sources/Recorder/Utilities/UIImage+SessionReplay.swift

```diff
@@ -7,60 +7,129 @@
 #if os(iOS)
 import Foundation
 import UIKit
+
 import DatadogInternal
-import CryptoKit
+import CommonCrypto
 
-private var srIdentifierKey: UInt8 = 11
+private let bitsPerComponent = 8
+private var identifierKey: UInt8 = 0
 
 extension UIImage: DatadogExtended {}
+
 extension DatadogExtension where ExtendedType: UIImage {
-    var srIdentifier: String {
-        if let hash = objc_getAssociatedObject(type, &srIdentifierKey) as? String {
-            return hash
-        } else {
-            let hash = computeHash()
-            objc_setAssociatedObject(type, &srIdentifierKey, hash, .OBJC_ASSOCIATION_RETAIN)
+    var identifier: String {
+        if let hash = objc_getAssociatedObject(type, &identifierKey) as? String {
             return hash
         }
+
+        let hash = md5Digest() ?? "\(type.hash)"
```

Comment:
Should we keep the previous hash function as a fallback in case a `UIImage` doesn't have a `cgImage`?

Reply:
Yes, just to prevent redoing the computation if it failed the previous time.